Bottom: d571b36b3901d5273ed83f1529d098f7d3123481
Top:    5078c33773a9a28bd8e031580adbe013b507ad21
Author: Jon Smirl <jonsmirl@gmail.com>
Date:   2009-05-23 13:31:59 -0400

Refresh of mpc5200-dma

---

diff --git a/sound/soc/fsl/mpc5200_psc_i2s.c b/sound/soc/fsl/mpc5200_psc_i2s.c
index b1c98eb..2186f4b 100644
--- a/sound/soc/fsl/mpc5200_psc_i2s.c
+++ b/sound/soc/fsl/mpc5200_psc_i2s.c
@@ -22,10 +22,6 @@
 #include "mpc5200_dma.h"
 #include "mpc5200_psc_i2s.h"
 
-MODULE_AUTHOR("Grant Likely <grant.likely@secretlab.ca>");
-MODULE_DESCRIPTION("Freescale MPC5200 PSC in I2S mode ASoC Driver");
-MODULE_LICENSE("GPL");
-
 /**
  * PSC_I2S_RATES: sample rates supported by the I2S
  *
@@ -295,4 +291,7 @@ static void __exit psc_i2s_exit(void)
 }
 module_exit(psc_i2s_exit);
 
+MODULE_AUTHOR("Grant Likely <grant.likely@secretlab.ca>");
+MODULE_DESCRIPTION("Freescale MPC5200 PSC in I2S mode ASoC Driver");
+MODULE_LICENSE("GPL");
